app/views/base/layout.scala.html

```diff
@@ -111,7 +111,7 @@
           <span data-icon="u">&nbsp;@langName(lang)</span>
         </a>
         <form action="@routes.I18n.select" method="POST" class="language_links dropdown" data-url="@staticUrl("trans/refs.json")?v=@assetVersion">
-          <li><a href="@routes.I18n.contribute">Help translate Lichess!</a></li>
+          <li><a href="@routes.I18n.contribute">@trans.helpTranslateLichess()</a></li>
```

Comment:
This must not be translated. If someone can't read english, they have no business in contributing translations